modules/transport-netty4/src/javaRestTest/java/org/elasticsearch/rest/Netty4BadRequestIT.java

```diff
@@ -63,7 +64,7 @@ public void testBadRequest() throws IOException {
         );
         assertThat(e.getResponse().getStatusLine().getStatusCode(), equalTo(BAD_REQUEST.getStatus()));
         assertThat(e, hasToString(containsString("too_long_http_line_exception")));
-        assertThat(e, hasToString(matches("An HTTP line is larger than \\d+ bytes")));
+        assertThat(e, hasToString(matchesRegex(Pattern.compile(".*An HTTP line is larger than \\d+ bytes.*", Pattern.DOTALL))));
```

Comment:
Is the exception message not a single line?

Reply:
The message has got some formatted JSON in it, with newlines. `matches()` requires a match against the entire string, and by default `.*` doesn't match newlines.